robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -188,6 +204,98 @@ function initRosWeb () {
 }
 
 /* functions used in main code */
+
+function getTimestamp () {
```

Comment:
since this is to return a specific ROS formatting of time. Then I suggest we call it either `rosTimeNow()` or `rosTimestamp()`